entity-service-impl/src/main/java/org/hypertrace/entity/data/service/EntityDataServiceImpl.java

```diff
@@ -694,6 +720,58 @@ private JSONDocument convertEntityRelationshipToDocument(
     }
   }
 
+  private void sentDeleteNotification(String tenantId, java.util.Collection<Entity> entities) {
+    try {
+      entityChangeEventGenerator.sendDeleteNotification(entities);
+    } catch (Exception ex) {
+      LOG.warn(
+          "For tenantId {] unable to send delete notification for entities {}",
```

Comment:
typo on the placeholder, but I'd encourage dropping the tenantID arg here. It's already part of the entity and thus is duplicate (and even if it weren't we could look it up if needed).

Reply:
Fixed